frontend/src/chrome/UsernameWithIcon.tsx

```diff
@@ -4,17 +4,19 @@ import classNames from 'classnames'
 interface UsernameWithIconProps {
   username: string
   className?: string
+  iconWidth?: number
 }
 
 // TODO(chriswhong): make the prop a user object, or pass in icon URL as a separate prop
 const UsernameWithIcon: React.FunctionComponent<UsernameWithIconProps> = ({
   username,
-  className
+  className,
+  iconWidth = 18
 }) => (
-   <div className={classNames('flex items-center text-xs tracking-wider leading-snug', className)}>
+   <div className={classNames('flex items-center tracking-wider leading-snug', className)}>
      <div className='rounded-xl inline-block mr-1 bg-cover flex-shrink-0' style={{
-       height: '18px',
-       width: '18px',
+       height: `${iconWidth}px`,
+       width: `${iconWidth}px`,
```

Comment:
provide `iconWidth` directly as the value, react will convert this to pixels for you